### ISSUE TITLE
Object detection

### ISSUE THREAD

**Issue by Jogima-cyber** [google-research/deeplab2#27]
I was wondering if this library offers a pre-trained model for object detection (I mean by object detection outputting the four coordinates of the detected boxes, and there is an indeterminate number of boxes)?

**Comment by aquariusjay** [comment]
Hi Jogima-cyber,

Thanks for the question.
In this library, we do not focus on object detection, which outputs the four bounding box coordinates.
Our library focuses more on the segmentation perspective, i.e., assigning each pixel with some labels (e.g., panoptic label).
You could infer the bounding box coordinates on top of the predicted masks by yourselves.
Additionally, if you really want to do object detection, you may want to refer to the following libraries:

- [TF Object Detection API](https://github.com/tensorflow/models/tree/master/research/object_detection)
- [TF Vision Model Garden](https://github.com/tensorflow/models/blob/master/official/vision/beta/MODEL_GARDEN.md)

Cheers,

**Comment by aquariusjay** [comment]
Closing the issue and hope you have found the library you need for object detection.
